moto/cognitoidp/exceptions.py

```diff
@@ -42,3 +43,11 @@ def __init__(self, message):
         self.description = json.dumps(
             {"message": message, "__type": "NotAuthorizedException"}
         )
+
+
+class InvalidParameterException(JsonRESTError):
```

Comment:
Existing error class extends  "BadRequest".
But it will be return http status code for  error code.
Error code should represent type of exception class.
I think  we should not use "BadRequest" class